packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -784,14 +780,7 @@ class ReactDOMServerRenderer {
   contextValueStack: Array<any>;
   contextProviderStack: ?Array<ReactProvider<any>>; // DEV-only
 
-  uniqueID: number;
-  identifierPrefix: string;
-
-  constructor(
-    children: mixed,
-    makeStaticMarkup: boolean,
-    options?: ServerOptions,
```

Comment:
niiiice

Reply:
Well we'll still have this option for Fizz. The motivation is so you can have multiple server rendered roots on the same page. Implementing for useId is next on the backlog.

I just chose not implement it for useId in the partial renderer because Fizz is about to roll out internally and the partial renderer doesn't exist in open source any more.